tests/foreman/sys/test_fam.py

```diff
@@ -74,6 +74,13 @@ def setup_fam(module_target_sat, module_sca_manifest, install_import_ansible_rol
     # Edit Makefile to not try to rebuild the collection when tests run
     module_target_sat.execute(f"sed -i '/^live/ s/$(MANIFEST)//' {FAM_ROOT_DIR}/Makefile")
```

Comment:
Not a blocker, but could we introduce an environment variable for this as well so we can avoid patching code altogether?

Reply:
Maybe, probably. But not today ;)